qemu/tests/cfg/hotplug_unplug_with_viommu.cfg

```diff
@@ -0,0 +1,21 @@
+- hotplug_unplug_with_viommu:
+    virt_test_type = qemu
+    type = nic_hotplug
+    required_qemu= [7.0.0,)
+    only q35
+    only Linux
+    only virtio_net
+    login_timeout = 360
+    clone_master = yes
+    master_images_clone = image1
+    remove_image_image1 = yes
+    repeat_times = 1
+    virtio_iommu = yes
+    pci_model = virtio-net-pci
+    netdev_extra_params_hotplug_nic1 = ",vhost=on"
```

Comment:
`vhost = on`?

Reply:
> `vhost = on`?

Let me check it again. I thought that vhost =on would not affect the parameters of hotplug, so I added this line of configuration.

Reply:
> > `vhost = on`?
> 
> Let me check it again. I thought that vhost =on would not affect the parameters of hotplug, so I added this line of configuration.

@PaulYuuu I tested it without this line, according to test reuslt, if this line be removed, hotplug command will not include vhost. So I think we should keep it.Please correct me if I'm wrong.
`{'execute': 'netdev_add', 'arguments': {'type': 'tap', 'id': 'idRqG4TJ', 'fd': '91'}, 'id': 'YNB1Q184'}`